recipes/pylama/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pylama" %}
+{% set version = "7.3.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "b5a8133d953798e00f96081cb77d2aa63c7cdab79a77a000fb51891887e8dabd" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - pylama = pylama.main:shell
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - mccabe >=0.5.2
+    - pycodestyle >=2.0.0
+    - pydocstyle >=1.1.1
+    - pyflakes >=1.3.0
+
+test:
+  imports:
+    - pylama
+    - pylama.libs
+    - pylama.lint
+
+  commands:
+    - pylama --help
+
+about:
+  home:  https://github.com/klen/pylama
+  license: LGPL 3.0
+  license_family: LGPL
```

Comment:
It looks like the `license_file` should be available in the src. Please add it when you get a chance. Otherwise LGTM.